app/Models/User.php

```diff
@@ -51,4 +52,15 @@ class User extends Authenticatable
     ];
 
     protected $table = 'appuser';
+
+    public function isVocabularyEditor() {
+        $userId = $this->id;
+        $vocabularyEditorId = '83580835';
```

Comment:
Let's hardcode 'Edit Vocabulary' permission label rather than the ID, as the ID can vary across environments.